README.md

```diff
@@ -1,73 +1,243 @@
+
+- [Installation](https://api.readme.dev/docs/installation)
+- [Usage](#usage)
+  - [OpenAPI definitions](#openapi-definitions)
+    - [General](#general)
+    - [Operations](#operations)
+    - [Servers](#servers)
+    - [Specification Extensions](#specification-extensions)
+    - [User Authentication](#user-authentication)
+  - [Operations](#operations-1)
+    - [General](#general-1)
+    - [Callbacks](#callbacks)
+    - [Request Body](#request-body)
+    - [Responses](#responses)
+    - [Security](#security)
+    - [Specification Extensions](#specification-extensions-1)
+  - [Callbacks](#callbacks)
+    - [General](#general-2)
+  - [Webhooks](#webhooks)
+- [FAQ](#faq)
 
 ## Installation
 
 ```
 npm install oas
 ```
 
-## CLI
-The CLI tool makes creating API definition files easier. It currently supports [OpenAPI 3.x](https://swagger.io/specification/) and [Swagger 2.0](https://swagger.io/specification/v2/) documents.
+## Usage
 
-### Usage
+> **Note**
+> If you need to use this library within a browser you'll likely need to use a bundler like [Webpack](https://webpack.js.org/) or [Rollup](https://rollupjs.org/).
 
-Go to a directory with your API, and type:
+`oas` offers a main `Oas` class, which will be your main entrypoint for using the library.
 
-```
-oas init
+```js
+import Oas from 'oas';
+import petstoreSpec from '@readme/oas-examples/3.0/json/petstore.json';
+
+const petstore = new Oas(petstoreSpec);
 ```
 
-It will walk you through how to document your API with a OpenAPI 3.0 Spec.
+Here the `Oas` constructor takes a JSON API definition (`petstoreSpec`). All API definitions you feed it must be JSON and must be an OpenAPI definition. If you have a YAML or Swagger definition you will need to convert it (our [oas-normalize](https://npm.im/oas-normalize) library can do this for you). And if you're in a CJS or non-`import` environment you can pull the library in with `require('oas').default`.
 
-#### Commands
+From here, the following APIs are at your disposal.
 
-```
-Usage: oas <command>
+### OpenAPI definitions
 
-  $ oas init                    Create a new OpenAPI definition
-  $ oas help                    Learn what you can do with oas
-  $ oas endpoint                Learn how to document an endpoint
-  $ oas generate [oas.json]     Output your OpenAPI definition (use --pretty for colors)
-  $ oas validate [oas.json]     Validate your OpenAPI definition
-```
+Because this library has full TypeScript types and docblocs this README is not intended to be full documentation so consult the individual method docblocks if you need more information on a specific method.
```

Comment:
```suggestion
Because this library has full TypeScript types and docblocks this README is not intended to be full documentation so consult the individual method docblocks if you need more information on a specific method.
```